.changeset/soft-berries-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-astro': patch
+---
+
+Skip cleanup after template download failed if the path is `.`, `./` or starts with `../`.
```

Comment:
```suggestion
No longer attempts to delete the directory after a template download fails if the path is `.`, `./` or starts with `../`.
```
Just wondering what you think about an example like this, @Genteure -- if these changelog messages are read by users, what about something more descriptive of the problem seen itself? (I'm inferring this from reading the attached issue. Make the statement correct as necessary!)

Your call, since your change does describe at a technical level what has changed in the code. But often these are meant to be user-facing "docs" of a sort describing effects people *using* the packages will see. If people want to know exactly how the code achieved that, they can look at the PR. So, it's an option to consider!